sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -167,599 +153,6 @@ case class DescribeDatabaseCommand(
   }
 }
```

Comment:
The remaining commands are database only. We can move the database related commands to there.